mw-webapp/src/logic/wayPage/WayPage.module.scss

```diff
@@ -1,4 +1,22 @@
+@import "src/styles/variables.scss";
+
 .container {
   width: 100%;
   margin: 0 auto;
 }
+
+.titlePage {
+  position: absolute;
+  top: 1rem;
+  left: 47%;
+  color: $lightyellow;
+  text-shadow: 2px 2px 2px $button-shadow;
+}
+
+.titleWhenLoggedIn {
+  position: absolute;
+  top: 2.2rem;
+  left: 47%;
+  color: $lightyellow;
+  text-shadow: 2px 2px 2px $button-shadow;
+}
```

Comment:
Its a bad idea to use so many styles for title on each page

Reply:
I made mixins